serde-generate/tests/dart_runtime.rs

```diff
@@ -0,0 +1,38 @@
+use serde_generate::{dart, test_utils, CodeGeneratorConfig, Encoding, SourceInstaller};
+use std::{io::Result, path::PathBuf, process::Command};
+use tempfile::tempdir;
+
+fn install_test_dependency(path: PathBuf) -> Result<()> {
+    Command::new("dart")
+        .current_dir(path)
+        .args(["pub", "add", "-d", "test"])
+        .status()?;
+
+    Ok(())
+}
+#[test]
+fn test_dart_runtime() {
```

Comment:
So we are quite testing interoperability of serialization formats here. That's fine but let's document the language as "in progress".